.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
We can add a section for use cases.

Reply:
This should be the first section. We can change the wording if it's not clear enough.

Reply:
Ok, the first section works :)